packages/backend/src/payment-method/ilp/spsp/middleware.test.ts

```diff
@@ -21,6 +25,7 @@ describe('SPSP Middleware', (): void => {
 
   beforeEach((): void => {
     next = jest.fn()
+    spspMiddleware = createSpspMiddleware(true)
```

Comment:
can we test the `false` path here?

Reply:
You mean that next is called without anything happening?

Reply:
Yes, a boring but good test to have